youtube-gpt-client/src/pages/how-it-works.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Grid, GridItem, Heading, Text } from '@chakra-ui/react';
+import { GetStaticProps } from 'next';
+import { useTranslation } from 'react-i18next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { CSSProperties } from 'react';
+
+const indexDotStyle: CSSProperties = {
+  content: '""',
+  display: 'inline-block',
+  width: '8px',
+  height: '8px',
+  borderRadius: '4px',
+  backgroundColor: '#5893CE',
+  marginLeft: '5px',
+};
+
+const HowItWorks = () => {
+  const { t } = useTranslation('howItWorks');
+  const features = [
```

Comment:
What are your thoughts on making this a service function? This way, we keep the page to only the UI and receive the data from a service using something like ```const features = get....();``` even if it is a mock.

Reply:
I'm on it, thanks for the sugestion!